deepspeed/module_inject/auto_tp.py

```diff
@@ -306,6 +306,8 @@ def tp_parser(model):
                 # Mixtral-7x8b used w2*act(w1*w3) linear. need to replace w2 to linearallreduce.
                 elif 'w2' in layer and 'Mixtral' in str(type(module)):
                     gem_list = gem_list + [layer]
+                elif "self_attn.dense" in layer:
```

Comment:
Should this also have a check for `and "Phi" in str(type(module))`?

Reply:
Added it~